vscode_extension/src/outputChannel.ts

```diff
@@ -0,0 +1,24 @@
+import { window, OutputChannel } from "vscode";
+
+let _extension_output_channel: OutputChannel;
+
+/**
+ * @returns vscode.OutputChannel for AaC extension output to the VSCode Output Panel.
+ *
+ * @example
+ * Sets the content of an output panel to `Hello!`:
```

Comment:
I'm referencing the panel component of the VSCode IDE, since that's what the output channel pipes to/creates. https://code.visualstudio.com/api/extension-capabilities/extending-workbench